src/Symfony/Bridge/PhpUnit/ClockMock.php

```diff
@@ -27,6 +35,11 @@ public static function withClockMock($enable = null)
         self::$now = is_numeric($enable) ? (float) $enable : ($enable ? microtime(true) : null);
     }
 
+    /**
+     * Return current Unix timestamp.
```

Comment:
We are using `Returns` everywhere in phpdocs for description, so this needs to be changes everywhere here in this PR.